features/fields.feature

```diff
@@ -60,19 +52,19 @@ Functionaliteit: Aanpasbare representatie met de fields parameter
   Scenario: Slechts één enkel attribuut wordt gevraagd
     Als een ingeschreven persoon wordt geraadpleegd met fields=geslachtsaanduiding
     Dan wordt attribuut geslachtsaanduiding teruggegeven
-    En wordt geen enkel ander attribuut dan geslachtsaanduiding teruggegeven
-    En wordt geen enkele relatie van de resource in _links teruggegeven
     En wordt attribuut _links.self teruggegeven
+    En wordt geen enkel ander attribuut dan geslachtsaanduiding teruggegeven
```

Comment:
Ik heb een gevoel dat dit een vorm van pleonasme is. Kunnen we niet gewoon zeggen:

Dan wordt **alleen** attribuut geslachtsaanduiding teruggegeven
En wordt **alleen** attribuut _links.self teruggegeven
En is _embedded leeg